.github/workflows/static-analyze.yml

```diff
@@ -126,4 +126,4 @@ jobs:
           cd ..
 
       - name: "analyze php code"
-        run: php -d memory_limit=-1 composer --working-dir=./.build psalm -- --shepherd --show-info=false --output-format=checkstyle | cs2pr
+        run: php -d memory_limit=-1 composer --working-dir=./.build psalm -- --shepherd --show-info=false --output-format=checkstyle 
```

Comment:
Psalm with activated shepherd will only work starting with the next release
https://github.com/vimeo/psalm/pull/2779